src/shared/services/user.service.ts

```diff
@@ -0,0 +1,53 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable, signal } from '@angular/core';
+import { Observable } from 'rxjs';
+import { environment } from '../../enviroments/enviroment';
+import { User } from '../interfaces/User';
+
+const { baseUrl, accessToken } = environment;
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UserService {
+  constructor(private http: HttpClient) {}
+
+  //  API CALLS ----------
+  getOrganizersForApproval(): Observable<User[]> {
+    // Login from postman and set your access token from enviroment. We dont have login yet.
+    const headers = new HttpHeaders({
+      'Content-Type': 'application/json',
+      'X-Authorization': accessToken,
+    });
+
+    const options = { headers: headers };
+
+    return this.http.get<User[]>(
+      `${baseUrl}/user/organizersForApproval`,
+      options
+    );
+  }
+
+  // API CALS END-----
+
+  // SIGNALS --------
+
+  // Organizers for approval
+
+  organizersForApproval = signal<User[]>([]);
+
+  hasOrganizersForapproval: boolean = this.organizersForApproval().length < 1;
+
+  setOrganizersForApprove() {
```

Comment:
get, not set

Reply:
I have two different functions, the first in the api call is getOrganizers the second function setOrganizers is then we need to set them in the state.